dev/bots/test.dart

```diff
@@ -146,6 +146,7 @@ Future<void> main(List<String> args) async {
       'customer_testing': customerTestingRunner,
       'analyze': analyzeRunner,
       'fuchsia_precache': fuchsiaPrecacheRunner,
+      'snippets': _runSnippetsTests,
```

Comment:
Do the CI runners auto-pick up this new shard? Or do we need to configure CI somewhere to run this?